contrib/codegen-tools/codegen/src/main/ops/org/nd4j/codegen/ops/SDBaseOps.kt

```diff
@@ -588,6 +621,63 @@ fun SDBaseOps() =  Namespace("BaseOps"){
         useMixin(keepDimsDoc)
     }
 
+    Op("whereNumpy") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.controlflow"
+        javaOpClass = "WhereNumpy"
+        Input(NUMERIC, "x") { description = "The first array" }
```

Comment:
@treo we've always had 2 where implementations: I'm exposing both here and letting user pick.